dell-dss-9500M/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Dell DSS 9500M",
+  "rules": [
+    {
+      "path": "ipmi-fru.Builtin FRU Device (ID 0).Board Mfg",
+      "contains": "DELL"
+    },
+    {
+      "path": "ipmi-fru.Builtin FRU Device (ID 0).Board Part Number",
+      "contains": "0G50D4A00"
```

Comment:
Typically A00 should be a hardware version information, you should confirm and avoid version information in SKU rule, otherwise SKU will be invalid with hardware upgrading.
BTW, is 9500M a released product?